README.md

```diff
@@ -8,6 +8,7 @@ their [website rocks](http://jekyllrb.com/).
 ```
 git clone git@github.com:artsy/artsy.github.io.git
 cd artsy.github.io
+bundle
```

Comment:
`bundle exec bootstrap` will fail if you don't already have the correct versions referenced in `Gemfile.lock`.